x-pack/test/saved_object_api_integration/common/suites/bulk_get.ts

```diff
@@ -6,11 +6,12 @@
  */
 
 import expect from '@kbn/expect';
-import { SuperTest } from 'supertest';
 import { SAVED_OBJECT_TEST_CASES as CASES } from '../lib/saved_object_test_cases';
 import { SPACES } from '../lib/spaces';
 import { expectResponses, getUrlPrefix, getTestTitle } from '../lib/saved_object_test_utils';
 import { ExpectResponseBody, TestCase, TestDefinition, TestSuite } from '../lib/types';
+import { FtrProviderContext } from '../ftr_provider_context';
```

Comment:
For things only used as a type it's my understanding that it's better to explicitly import it as such. This also ensures that nothing is left behind at run-time, which could happen in certain scenarios:

```suggestion
import type { FtrProviderContext } from '../ftr_provider_context';
```

Reply:
Ah good call! Fixed in latest commit!